x-pack/plugins/fleet/server/plugin.ts

```diff
@@ -332,8 +333,33 @@ export class FleetPlugin
 
     this.telemetryEventsSender.start(plugins.telemetry, core);
 
+    const logger = appContextService.getLogger();
+
+    const fleetSetupPromise = (async () => {
+      try {
+        logger.info('Beginning fleet setup');
+
+        const { nonFatalErrors } = await setupFleet(
+          new SavedObjectsClient(core.savedObjects.createInternalRepository()),
+          core.elasticsearch.client.asInternalUser
+        );
+
+        if (nonFatalErrors.length > 0) {
+          logger.info('Encountered non fatal errors during Fleet setup');
+          formatNonFatalErrors(nonFatalErrors).forEach((error) =>
+            logger.info(JSON.stringify(error))
+          );
+        }
+
+        logger.info('Fleet setup completed');
+      } catch (error) {
+        logger.warn('Fleet setup failed');
+        logger.warn(error);
+      }
+    })();
```

Comment:
Should we move this logging logic into `setupFleet` itself so we get the same logging when running manually via the API?

Reply:
Yes I think that's a good idea.

Reply:
I moved all the logs except for what's in our `catch` into `setupFleet` in https://github.com/elastic/kibana/pull/117552/commits/76138ec1c31c57ef08373e3f5361dc25b24a5dd7